bugbash.md

```diff
@@ -1,2 +1,7 @@
 # Bug Bash File For Activity
 Who goes first?
+
+## I Can Make Next PR
+- hamburgers
```

Comment:
changed order of food

Reply:
nice 👍 - we should add one more if you can think of any

Reply:
ok yeah that makes sense